src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import { Header, Footer } from "./layout";
+import { Welcome, User, Repo } from "./pages";
+import './index.css';
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <Switch>
+        <Route exact path="/">
+          <Welcome />
+        </Route>
+        <Route exact path="/user">
```

Comment:
This route returns a 404 when a user refreshes. Maybe you could use a 'username' param, so that it can fetch based on that?